.github/workflows/component-builder.yml

```diff
@@ -41,19 +41,22 @@ jobs:
         working-directory: ./containers/fedora
         env:
           PYTHONPATH: ${{ github.workspace }}
-          ORGANIZATION_ID: ${{ secrets.BITWARDEN_ORGANIZATION_ID }}
-          ACCESS_TOKEN: ${{ secrets.BITWARDEN_ACCESS_TOKEN }}
+          NO_SECRETS: "true"
         run: ./build.sh
-      - name: Logging to quay.io
-        run: podman login -u ${{ secrets.QUAY_USER }} -p ${{ secrets.QUAY_TOKEN }} quay.io
-      - name: Tag & Push image to staging
+      - name: Save container image as tarball
         env:
           local_repository: "localhost/fedora"
           remote_repository: "quay.io/openshift-cnv/qe-cnv-tests-fedora-staging"
           arch_tag: "${{ env.FEDORA_VERSION }}-${{ env.CPU_ARCH }}"
-          remote_tag: "${{ env.FEDORA_VERSION }}-dev"
         run: |
+          mkdir -p artifacts
           podman tag "${local_repository}":"${arch_tag}" "${remote_repository}":"${arch_tag}"
-          podman push "${remote_repository}":"${arch_tag}"
-          podman manifest create --log-level=debug "${remote_repository}":"${remote_tag}" "${remote_repository}":"${arch_tag}"
-          podman manifest push "${remote_repository}":"${remote_tag}" --all --format=v2s2
+          podman save -o artifacts/fedora-image.tar "${remote_repository}":"${arch_tag}"
+          echo "Saved image to artifacts/fedora-image.tar"
+      - name: Upload container image artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: fedora-container-image
+          path: artifacts/fedora-image.tar
```

Comment:
how can this be used by a user? can you add some explanation here (i will move it to the updated readme)

Reply:
A user can download the artifact and load the image into the local image storage.

Steps:
1. Access the action workflow on github: https://github.com/RedHatQE/openshift-virtualization-tests/actions/workflows/component-builder.yml
2. Click on the relevant run, e.g. : https://github.com/RedHatQE/openshift-virtualization-tests/actions/runs/15147820347
3. On the bottom of the page, click on the artifact to download.
4. Once on the local storage, extract the tar file from the zip.
5. Load the image into the local image storage using `podman`:
`podman load -i fedora-image.tar`